fe/fe-core/src/test/resources/conf/config_test3.properties

```diff
@@ -0,0 +1 @@
+schedule_slot_num_per_path_only_for_test = 5
```

Comment:
This test alias can be removed in production code paths.

Reply:
i will fix this